src/crepe/facade/SDLContext.h

```diff
@@ -1,48 +1,160 @@
 #pragma once
 
+#include <SDL2/SDL_keycode.h>
 #include <SDL2/SDL_render.h>
 #include <SDL2/SDL_video.h>
 
 #include "../api/Sprite.h"
 #include "../api/Transform.h"
-#include "../system/RenderSystem.h"
+#include "api/Camera.h"
+
+typedef SDL_Keycode CREPE_KEYCODES;
```

Comment:
Deze typedef is nu niet gedefiniëerd in de `crepe` namespace, en is niet bruikbaar vanuit de API omdat het onderliggende type nog steeds door SDL2 gedefiniëerd is